website/data/docs-nav-data.json

```diff
@@ -471,7 +471,7 @@
                 "path": "connect/gateways/mesh-gateway"
               },
               {
-                "title": "WAN Federation",
+                "title": "Enabling WAN Federation Control Plane Traffic",
```

Comment:
I like this new name a lot. Optional: do you think we should change the page H1 title as well? I think yes.

Reply:
Updated the title in a subsequent commit. Hope it makes sense to you.